sql/testdata/txn

```diff
@@ -152,3 +152,34 @@ COMMIT TRANSACTION
 
 statement error there is no transaction in progress
 ROLLBACK TRANSACTION
+
+# Set isolation level without a transaction is an error.
+
+statement error there is no transaction in progress
+SET TRANSACTION ISOLATION LEVEL SNAPSHOT
+
+statement ok
+BEGIN TRANSACTION ISOLATION LEVEL SNAPSHOT; COMMIT
+
+statement ok
+BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE; COMMIT
+
+statement ok
+BEGIN TRANSACTION; SET TRANSACTION ISOLATION LEVEL SNAPSHOT; COMMIT
+
+statement ok
+BEGIN TRANSACTION; SET TRANSACTION ISOLATION LEVEL SERIALIZABLE; COMMIT
+
+# It is an error to change the isolation level of a running transaction.
+
+statement ok
+BEGIN TRANSACTION
+
+statement ok
+UPDATE kv SET v = 'b' WHERE k in ('a')
+
+statement error cannot change the isolation level of a running transaction
+SET TRANSACTION ISOLATION LEVEL SNAPSHOT
+
+statement ok
+ROLLBACK
```

Comment:
I've added support for `SHOW TRANSACTION ISOLATION LEVEL` and added a few test cases to `testdata/txn`.